RiotSwiftUI/Modules/Room/RoomAccess/Coordinator/RoomAccessCoordinatorBridgePresenter.swift

```diff
@@ -34,6 +35,7 @@ final class RoomAccessCoordinatorBridgePresenter: NSObject {
     // MARK: Private
     
     private let room: MXRoom
+    private let parentId: String?
```

Comment:
To keep consistency, this should probably be `parentSpaceId` everywhere, if we are referring to a space from a room (as opposed to from child space to a parent space.

An alternative would be to introduce some `SpaceIdentifier` typealias so that it is always clear what the parent points to. But in this case that would not be objc compatible.

Reply:
I changed the name but I prefer to keep it simple and objc compatible for the moment.